modules/container-instance/variables.tf

```diff
@@ -0,0 +1,109 @@
+variable "container_image" {
+  type        = string
+  description = "Image of the container"
+}
+
+variable "container_instance_name" {
+  type        = string
+  description = "Name of the container instance"
+}
+
+variable "container_name" {
+  type        = string
+  description = "Name of the container"
+}
+
+variable "container_registry_login_server" {
+  type        = string
+  description = "Login server of the container registry"
+}
+
+variable "location" {
+  type        = string
+  description = "Azure region where the resource should be deployed."
+  nullable    = false
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "Name of the resource group"
+}
+
+variable "user_assigned_managed_identity_id" {
+  type        = string
+  description = "ID of the user-assigned managed identity"
+}
+
+variable "availability_zones" {
+  type        = list(string)
+  default     = [1]
```

Comment:
We should default to zonal if possible [1, 2, 3]

Reply:
Not possible for container groups, they only support a single zone. The code places them across multiple zones.